tests/classification/test_auc.py

```diff
@@ -32,19 +32,29 @@ def sk_auc(x, y):
     return _sk_auc(x, y)
 
 
+def sk_auc_reorder(x, y):
+    x = x.flatten()
+    y = y.flatten()
+    idx = np.argsort(x, kind='stable')
+    x = x[idx]
+    y = y[idx]
+    return _sk_auc(x, y)
```

Comment:
```suggestion
def sk_auc(x, y, reorder=False):
    x = x.flatten()
    y = y.flatten()
    if reorder:
        idx = np.argsort(x, kind='stable')
        x = x[idx]
        y = y[idx]
    return _sk_auc(x, y)
```
follow https://github.com/PyTorchLightning/metrics/pull/197/files#r619033509